### ISSUE TITLE
plugin does not work with docker.

### ISSUE THREAD

**Issue by yabloki** [robincsamuel/ghost-google-drive#28]
I get this error no matter what i do:
`Unable to find storage adapter ghost-google-drive in ,/var/lib/ghost/content/adapters/,/var/lib/ghost/versions/3.15.3/core/server/adapters/.`

My Dockerfile includes next steps:
> cd /var/lib/ghost/versions/3.15.3/core/server/adapters/storage ;\
	git clone https://github.com/robincsamuel/ghost-google-drive.git ;\
	cd ghost-google-drive ;\
	npm install; \
	cd .. ;\
	chmod -R 666 ghost-google-drive ;\
	chown -R node:node ghost-google-drive ;\
	mkdir -p  /var/lib/ghost/content/adapters/storage ;\
	cp -R ghost-google-drive/ /var/lib/ghost/content/adapters/storage ;

config:
```
"storage": {
          "active": "ghost-google-drive",
          "ghost-google-drive": {
            "key": {
                    "private_key_id": "..",
                    "private_key":"..",
                    "client_email": "..",
                    "client_id": ".."
             }
          }
      },
"paths": {
      "contentPath": "/var/lib/ghost/content"
    }
```

Inside of the container If I run a node and do require, it works
```
require('/var/lib/ghost/content/adapters/storage/ghost-google-drive')
[Function: ghostGoogleDrive]
```
And I see the plugin in all the right places on the disk

1.
```
bash-5.0# cd /var/lib/ghost/content/adapters/storage/ghost-google-drive/
bash-5.0# ls
LICENSE            README.md          index.js           node_modules       package-lock.json  package.json
bash-5.0#
```
2.

```
bash-5.0# cd /var/lib/ghost/versions/3.15.3/core/server/adapters/storage/ghost-google-drive/
bash-5.0# ls
LICENSE            README.md          index.js           node_modules       package-lock.json  package.json
bash-5.0#
```
I see the error coming from AdapterManager.js:110, after require fail on AdapterManager.js:89. I have no ideas why it fails though and what to do next.

**Comment by robincsamuel** [comment]
Looks like your path is wrong, it starts with a comma, `,/var..`.

**Comment by yabloki** [comment]
No,  paths to look for adapter include blank path hence comma. (AdapterManager.js:89)
The problem was with permissions inside the container. ghost-entrypoint script changes ownership if run by root. Solution was to change entrypoint in dockerfile to run directly node current/index.js instead of docker-entrypoint.sh

**Comment by robincsamuel** [comment]
Ah okay. Glad you solved it :-)

**Comment by sgofferj** [comment]
@yabloki Would you share your Dockerfile? Or @robincsamuel Would you consider including an example to your documentation?
I'm currently trying to create a Dockerfile to include the plugin and set it up - but not very successfully...

**Comment by robincsamuel** [comment]
@sgofferj Sorry, I haven't tried making a Dockerfile with this plugin yet. It'd be great if @yabloki can share the working Dockerfile and I can include the same in the documentation.

But I can try to help if you can share more details about the problem you are facing.

**Comment by sgofferj** [comment]
@robincsamuel I have tried various methods of getting the connector installed through a dockerfile but most failed in one way or another. What got me the furthest was the example from the cloudinary connector but also this didn't work in the end. Everything I tried before trying the cloudinary method let to ghost not even starting. I just started tinkering with Docker a few months ago and I'm not very deep in nodejs/npm yet.
